src/services/userDetails.services.js

```diff
@@ -0,0 +1,19 @@
+import UserDetailsModel from '../database/models/userDetails.model';
+
+async function getUserDetailsById(userId) {
+  const userInfo = await UserDetailsModel.findOne({
+    where: { userId },
+  });
+  return userInfo;
+}
+async function createUserDetails(data) {
+  const userInfo = await UserDetailsModel.create(data);
+  return userInfo;
+}
+async function updateUserDetails(userId, data) {
+  const userInfo = await UserDetailsModel.update(data, {
+    where: { userId },
+  });
+  return userInfo;
+}
+export default { getUserDetailsById, createUserDetails, updateUserDetails };
```

Comment:
All these three functions are reusable, you can turn them into services, import the model from there only and call those functions in this file. As in we shouldn't import the model everytime we to change a user profile.

Reply:
they are already in services